app/src/test/java/com/antosik/benchproject/app/popular/movies/viewModel/PopularMoviesViewModelTest.kt

```diff
@@ -30,52 +29,31 @@ internal class PopularMoviesViewModelTest {
     )
     val getPopularMoviesUseCase: GetPopularMoviesUseCase = mockk()
     val popularMoviesFragmentNavigator: PopularMoviesFragmentNavigator = mockk()
+    val successState = UIState.Success(movies)
+    val loadingState = UIState.Loading
+    val emptyState = UIState.Empty
 
     @Test
```

Comment:
navigateToMovieDetailsFragment -> this method is not tested but it should be :)

Reply:
Done 😄